resolwe/flow/utils/test.py

```diff
@@ -438,7 +439,7 @@ def _debug_info(self, data):
         path = os.path.join(settings.FLOW_EXECUTOR['DATA_DIR'], str(data.pk), "stdout.txt")
         if os.path.isfile(path):
             msg += "\nstdout.txt:\n" + 11 * "-" + "\n"
-            with open(path, 'r') as fn:
+            with codecs.open(path, 'r', encoding='utf-8') as fn:
```

Comment:
I would prefer if we use `io.open`.

For more details see [*io.open vs. codecs.open*](https://mail.python.org/pipermail/python-list/2015-March/687124.html).